exonum-java-binding-core/src/main/java/com/exonum/binding/service/adapters/UserServiceAdapter.java

```diff
@@ -130,6 +135,7 @@ public UserTransactionAdapter convertTransaction(byte[] transactionMessage) {
       return service.initialize(fork)
           .orElse(null);
     } catch (CloseFailuresException e) {
+      LOGGER.error("Failed to close resources during initialize", e);
```

Comment:
It seems like a change from ECR-2651 — I'd make it in a separate PR, as it requires more changes — see the issue description.

Reply:
Currently, this exception **is** logged by the logger attached through AOP